core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -39,6 +39,15 @@ private TableProperties() {
   public static final String COMMIT_NUM_STATUS_CHECKS = "commit.num-status-checks";
   public static final int COMMIT_NUM_STATUS_CHECKS_DEFAULT = 3;
 
+  public static final String COMMIT_STATUS_CHECKS_MIN_WAIT_MS = "commit.status-check.min-wait-ms";
```

Comment:
@coolderli, @aokolnychyi, has `commit.num-status-checks` been released yet?

If not, then we should rename it to mirror the `commit.retry.*` settings: `commit.status-check.num-retries`